src/main/resources/application.properties

```diff
@@ -60,7 +62,7 @@ report.state.open=  ![#00ff00](https://via.placeholder.com/15/00ff00/000000?text
 report.state.old=  ![#ffaa00](https://via.placeholder.com/15/ffaa00/000000?text=+)
 report.state.ancient=  ![#ff0000](https://via.placeholder.com/15/ff0000/000000?text=+)
 
-report.cron.expr=0 0 12 ? * THU *
+report.cron.expr=0 30 13 ? * THU *
```

Comment:
What is the purpose of this property as we have another one `github.reporting.collector.cron.expr` ?

Reply:
I have replaced the default schedule, instead the report being generated at 12:00 it is now generated at 13:30 because a sync task will be executed at 12:00.

Reply:
So we have 2 jobs, one to collect / fetch the data and another to populate the report